### ISSUE TITLE
Cannot install logstash-filter-translate at Elastic Stack 5.0.0-alpha3?

### ISSUE THREAD

**Issue by demantos** [logstash-plugins/logstash-filter-translate#32]
- Version: logstash-5.0.0-alpha3
- Operating System: Ubuntu 14.04 LTS

I install Elastic Stack 5.0.0-alpha3
But logstash-filter-translate is not installed by default.
So, I try this

`root@ubuntu:/usr/share/logstash# bin/logstash-plugin install logstash-filter-translate`

But, it is not working
I try build gem file and install

```
root@ubuntu:/usr/share/logstash# bin/logstash-plugin install logstash-filter-translate/logstash-filter-translate-2.1.4.gem
Validating logstash-filter-translate/logstash-filter-translate-2.1.4.gem
Installing logstash-filter-translate
Plugin not found, aborting
ERROR: Installation Aborted, message: You have requested:
  logstash-core-plugin-api = 1.18.0

The bundle currently has logstash-core-plugin-api locked at 2.1.6.
Try running `bundle update logstash-core-plugin-api`
```

logstash-filter-translate is dependent by logstash-core-plugin-api 1.18.0
logstash-core-plugin-api 1.18.0 is dependent by logstash-core (<= 2.3.3, >= 2.0.0) java
but I using logstash-core (= 5.0.0.alpha3) java

So, I cannot install logstash translate plugin in my Elastic Stack 5.0.0-alpha3?

**Comment by shoggeh** [comment]
I totally support this. Filter translate is one of the most useful plugins in my scenerio and I'd love to see it working with alpha3+

Would you be so kind to make a release compatible with 2.0 plugin api?

**Comment by algestam** [comment]
This plugin has now been updated to be compatible with 5.0.0-alpha3+

**Comment by jsvd** [comment]
this has been fixed in #34

**Comment by shoggeh** [comment]
Hi,

An attempt to install this locally with clean logstash alpha5 instance is resulting with this on CentOS 6:

# rpm -qa java-\* logstash
logstash-5.0.0~alpha5-1.noarch
java-1.8.0-openjdk-headless-1.8.0.101-3.b13.el7_2.x86_64

# /usr/share/logstash/bin/logstash-plugin install --local /home/maciej/logstash-filter-translate-3.0.0.gem 
Validating /home/maciej/logstash-filter-translate-3.0.0.gem
Installing logstash-filter-translate
which: no javac in (/usr/local/sbin:/sbin:/usr/sbin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin:/root/bin)
Error Bundler::InstallError, retrying 1/10
An error occurred while installing logstash-core-event-java (5.0.0.pre.alpha5), and Bundler cannot continue.
Make sure that `gem install logstash-core-event-java -v '5.0.0.pre.alpha5'` succeeds before bundling.`
[...]

Is everything OK? Does it really need to pull javac as dependency?
